src/components/InfoViewer/formatters/common.ts

```diff
@@ -1,12 +1,13 @@
 import type {TDirEntry} from '../../../types/api/schema';
+import {EMPTY_DATA_PLACEHOLDER} from '../../../utils/constants';
 import {formatDateTime} from '../../../utils/dataFormatters/dataFormatters';
 import i18n from '../i18n';
 import {createInfoFormatter} from '../utils';
 
 export const formatCommonItem = createInfoFormatter<TDirEntry>({
     values: {
         PathType: (value) => value?.substring('EPathType'.length),
-        CreateStep: formatDateTime,
+        CreateStep: (value) => (Number(value) ? formatDateTime(value) : EMPTY_DATA_PLACEHOLDER),
```

Comment:
maybe it would be better to handle zero values in formatDateTime and to pass EMPTY_DATA_PLACEHOLDER as defaultValue ?

Can't imagine anyone would need 1970-01-01 03:00 value (that is returned for formatDateTime(0)) anywhere accross the project